charts/enterprise-kyverno-operator/Chart.yaml

```diff
@@ -2,8 +2,8 @@ apiVersion: v2
 name: nirmata-kyverno-operator
 description: Helm Chart for Enterprise Kyverno Operator
 type: application
-version: v0.3.10
-appVersion: v0.2.2
+version: v0.3.11
+appVersion: v0.2.3
```

Comment:
Not seeing a 0.2.3 version in kyverno-operator yet.

Reply:
Looks like resolving issues in the operator might take time. Going back to the old image for now.